WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController+Helpers.swift

```diff
@@ -67,4 +67,80 @@ extension ProductFormViewController {
         alertController.addAction(confirm)
         present(alertController, animated: true)
     }
+
+    /// Variation Deletion Confirmation alert
+    ///
+    func presentVariationConfirmationDeleteAlert(completion: @escaping (_ isConfirmed: Bool) -> ()) {
+        let title = NSLocalizedString("Remove variation",
+                                      comment: "Title of the alert when a user is deleting a variation")
+        let body = NSLocalizedString("Are you sure you want to remove this variation?",
+                                     comment: "Body of the alert when a user is deleting a variation")
+        let cancelButton = NSLocalizedString("Cancel",
+                                             comment: "Cancel button on the alert when the user is cancelling the action on deleting a variation")
+        let confirmButton = NSLocalizedString("Remove",
+                                              comment: "Confirmation button on the alert when the user is deleting a variation")
```

Comment:
Do you think we can move these strings(and all the others) to the `Localization` enum as we do on other view controllers?

Reply:
I felt that this extension is already feels like a list of strings, but extraction allowed to simplify a logic a bit and it looks cleaner now (dfbc904) 🎉